tests/components/ct-react/src/Button.spec.tsx

```diff
@@ -0,0 +1,13 @@
+import { test, expect } from '@playwright/test';
+import { Button } from './Button';
+
+test.use({ viewport: { width: 500, height: 500 } });
+
+test('should work', async ({ mount }) => {
+  let clicked = false;
+  const component = await mount(<Button onClick={() => clicked = true }></Button>);
+  await expect(component).toContainText('click me');
+  expect(clicked).toBe(false);
+  await component.click();
+  expect.poll(() => clicked).toBe(true);
```

Comment:
Did we figure out why it was not working during the demo?

Reply:
Per-discussion:
1. we should make this work without poll
2. we still need to resolve the complaint in the console that we saw this AM about circular objects